README.md

```diff
@@ -83,30 +83,30 @@ where:
 
 __Note:__ the input TSV file needs to consist of a single column named "ID".
 
-2) To scrape all run, study, BioProject and other IDs with associated DOI names from an existing web Zotero library collection into a `NCBIAccessionIDs` artifact run:
+2) To scrape all run and BioProject IDs from an existing web Zotero library collection into a `NCBIAccessionIDs` artifact run, you can use the `scrape-collection` method. Before running it, you have to set three environment variables linked to your Zotero account:
+* `TYPE` is the Zotero library type 'user' or 'group'.
```

Comment:
Is it really necessary to have the library type here? It's not sensitive information and we already have some other Zotero-related params anyway.

Reply:
yes, it's necessary as depending on the library type the USERID is different (see next line in README).

Reply:
Fine.